examples/3d-tiles/tile-3d-layer.js

```diff
@@ -289,13 +306,43 @@ export default class Tile3DLayer extends CompositeLayer {
     // tileHeader.userData = {gltfObjects};
   }
 
+  _calculatePositionsOffset(tileHeader) {
+    const {center} = tileHeader.boundingVolume;
+    const {distanceScales} = this.context.viewport;
+
+    const {metersPerPixel, pixelsPerDegree} = distanceScales;
+
+    const {pointsCount} = tileHeader.userData;
+    const {positions} = tileHeader.content;
+
+    for (let i = 0; i < pointsCount; i++) {
+      const point = new Vector3(positions[i * 3], positions[i * 3 + 1], positions[i * 3 + 2]);
+
+      Ellipsoid.WGS84.cartesianToCartographic(point, point);
```

Comment:
Converting each point to cartographic separately in JS looks computationally expensive. 

One alternative is to convert the boundingVolume center to cartesian (once) and use that as offset.

However, note that the numeric values of the point also depend on what transforms are being applied. You could apply the inverse of the current transform to the center.

But the goal is just to make the point coordinates in the array small.

So to make sure we don't have to worry about transforms, we could just calculate the average of the points in the array (without converting them to cartesian) and then just subtract that from all the points.

And then add the offset to our transform.